src/main/java/com/google/api/codegen/config/RetryCodesConfig.java

```diff
@@ -218,18 +227,39 @@ private void populateRetryCodesDefinitionWithProtoFile(
       }
 
       String retryCodesName;
+      ImmutableList<String> retryCodesList;
       if (protoParser.isHttpGetMethod(method)) {
+        retryCodesList = RETRY_CODES_FOR_HTTP_GET;
         // It is a common case to have an HTTP GET method with no extra codes to retry on,
         // so let's put them all under the same retry code name.
-        retryCodesName = httpGetRetryName;
-        retryCodesDefinition.put(httpGetRetryName, RETRY_CODES_FOR_HTTP_GET);
+        if (!new HashSet<>(RETRY_CODES_FOR_HTTP_GET)
```

Comment:
Won't this check return the same result for every method? If so, can we move it outside the loop?

Reply:
:D